demo/app.js

```diff
@@ -655,6 +657,7 @@ const Demo = React.createClass({
               Footer Content
             </div>
           )}
+          footerStyle={{ padding: '40px 20px' }}
```

Comment:
Normally we put it all at the bottom and calling it. The app.ls is the demo we use for testing changes and is a little unwieldy.